pkg/devfile/generator/generators_test.go

```diff
@@ -168,11 +170,14 @@ func TestGetContainers(t *testing.T) {
 			wantContainerName:  containerNames[0],
 			wantContainerImage: containerImages[0],
 			wantContainerEnv: []corev1.EnvVar{
-
 				{
 					Name:  "PROJECTS_ROOT",
 					Value: "/myroot",
 				},
+				{
+					Name:  "PROJECT_SOURCE",
```

Comment:
Why are we adding `PROJECT_SOURCE` here twice with different values?

Reply:
I believe the issue is that the value is not being read properly if it is the final item but if that's the case I think we could probably find a better solution than just duplicating the value. wdyt @michael-valdron @thepetk am I understanding this correctly?

Reply:
Better to resolve in favor of https://github.com/devfile/library/pull/220/files#r1717510608